.github/workflows/deploy-s3-bucket.yml

```diff
@@ -0,0 +1,51 @@
+name: deploy-s3-bucket-to-s3
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  compile-and-upload:
+    runs-on: ubuntu-latest
+    steps:
+      - name: clone
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          persist-credentials: false
+
+      - name: cache-node-modules
+        uses: actions/cache@v1
+        id: cache-node-modules
+        with:
+          path: node_modules
+          key: node-modules-${{ hashFiles('yarn.lock') }}
+
+      - name: Install npm packages
+        if: steps.cache-node-modules.outputs.cache-hit != 'true'
+        run: |
+          npm i
+
+      - name: Compile code
+        run: |
+          make compile
+
+      - name: Prepare code for deploy
+        run: |
+          make deploy
+
+      - name: Build source package
+        run: |
+          make source-package
+
+      - name: Upload source package to S3 bucket
+        uses: jakejarvis/s3-sync-action@v0.5.1
+        with:
+          args: --acl public-read --follow-symlinks --delete
+        env:
+          AWS_S3_BUCKET: "wm-jitsi-meet"
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          AWS_REGION: "eu-west-1"
+          SOURCE_DIR: "s3-package"
+          DEST_DIR: "${GITHUB_SHA}"
```

Comment:
https://github.community/t/how-to-get-just-the-tag-name/16241
replace `"${GITHUB_SHA}"` with `${GITHUB_REF##*/}`

Reply:
Ouhhhh this is really cool